### ISSUE TITLE
URLs are detected as possible spelling mistakes

### ISSUE THREAD

**Issue by lopopolo** [drahnr/cargo-spellcheck#269]
**Describe the bug**

A URL beginning with `https://` or `http://` is reported as a spelling mistake.

**To Reproduce**

Steps to reproduce the behaviour:

1. A file containing:
    ```rust
    /// https://github.com/ruby/ruby/blob/v2_6_3/lib/securerandom.rb#L135
    ///
    /// See https://github.com/artichoke/artichoke/issues/930 for rationale of this
    ///
    /// See https://github.com/artichoke/artichoke/pull/1969.
    fn main() {
        println!("Hello, world!");
    }
    ```
3. Run `cargo spellcheck`

**Expected behavior**

No spelling mistakes are reported.

**Screenshots**

```
$ cargo spellcheck
error: spellcheck(Hunspell)
  --> /Users/lopopolo/Downloads/xxx/src/main.rs:1
   |
 1 |  https://github.com/ruby/ruby/blob/v2_6_3/lib/securerandom.rb#L135
   |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |   Possible spelling mistake found.

error: spellcheck(Hunspell)
  --> /Users/lopopolo/Downloads/xxx/src/main.rs:3
   |
 3 |  See https://github.com/artichoke/artichoke/issues/930 for rationale of this
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |   Possible spelling mistake found.

error: spellcheck(Hunspell)
  --> /Users/lopopolo/Downloads/xxx/src/main.rs:5
   |
 5 |  See https://github.com/artichoke/artichoke/pull/1969.
   |      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |   Possible spelling mistake found.

```

**Please complete the following information:**
 - System: macOS Monterey 12.5
 - Obtained: `cargo install cargo-spellcheck`
 - Version: `cargo-spellcheck 0.11.3`

**Additional context**

**Comment by drahnr** [comment]
CommonMark/Markdown expects urls to be use as `<https://foo.bar>` or `[baz](https://foo.bar)` besides a few more variants. All of those are supported. I am reluctant to add a workaround to ignore these. Adding a specific fix suggestion would be very welcome though!

**Comment by drahnr** [comment]
Unless someone opens a PR with a tractable solution, I am going to close this. It's a nice to have after all. The spec clearly defines how links should be outlined.
